src/components/OSCALBackMatter.js

```diff
@@ -21,11 +21,9 @@ export const OSCALBackMatterCard = styled(Card)(
 `
 );
 
-// TODO: Remove workaround for missing media type
-// https://github.com/EasyDynamics/oscal-react-library/issues/512
 const getURLMediaType = (url) => {
-  const lastUrlPath = url.split("//").pop().split("/").pop();
-  return lastUrlPath.match(/\.[A-Za-z]{3,4}($|\?)/) || "Unknown";
+  const fileExtension = new URL(url).pathname.split(".").reverse()[0];
```

Comment:
```suggestion
  const fileExtension = new URL(url).pathname.split(".").pop();
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/pop

or

```suggestion
  const fileExtension = new URL(url).pathname.split(".").at(-1);
```